.github/workflows/draft_new_release.yml

```diff
@@ -63,12 +63,22 @@ jobs:
         run: |
           cargo set-version --package webapp ${{ github.event.inputs.version }}
 
-      - name: Commit changelog and manifest files
+      - name: Bump mobile/native Cargo.toml version
+        uses: stellar/binaries@v13
+        with:
+          name: cargo-edit
+          version: 0.11.6
+      - id: set-mobile-version
+        continue-on-error: true
+        run: |
+          cargo set-version --package mobile/native ${{ github.event.inputs.version }}
```

Comment:
❌ At least locally this does not work.
Correct would be
```suggestion
           cargo set-version --package native ${{ github.event.inputs.version }}
```

Reply:
🙈 I tested it locally with a different command. Thanks for the fix 🙂